docs/general/administration/hardware-selection.md

```diff
@@ -9,55 +9,61 @@ This page covers what you need to know in order to select appropriate hardware f
 
 ## TL;DR
 
-For a Jellyfin server, the following is recommended:
+Below is a list of recommended specs to run Jellyfin. Even though Jellyfin will work on even lower end hardware, the following specs are recommended for a good experience.
 
-### Normal Server
+### Shared Items
 
-- CPU (With dGPU): Intel Core i5-2300, AMD Ryzen 3 1200 or better. (Geekbench 6 Multicore 1500 or better). *CPU vendor will not have a meaningful impact on quality and/or speed of transcoding.*
-- CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better, excluding J and N series)
-- RAM: 8GB or more
-- Storage: 60GB SSD storage for Jellyfin files and transcoding cache.
-- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel is recommended over Nvidia. AMD and Apple Silicon are not recommended.
+Some component recommendations do not change regardless of the hardware configuration:
 
-:::note Intel "Atom" CPUs
+- Storage: 100GB SSD for Jellyfin files and transcoding cache. (Consider adding more on Windows 11)
+- Networking: Gigabit Ethernet Adapter or faster, WiFi or Powerline not recommended.
+- Internet Connection: At least 20mbps upload bandwidth (for remote access)
```

Comment:
this also is highly user depended and will not really fit most. 20mbps will be consumed pretty quickly if no bandwith limit is configured.

Reply:
I put in 20mb because that is a reasonable number to handle a single (maybe 2) 4k stream or a couple of 1080p streams. I will add in the bandwidth limit recommendation.